artsy/reaction
# [NavBar] Connect NotificationsMenu to relay

This replaces static layout data in the `NotificationsMenu` with a relay fetch. 

**Todo:**
- Finish mobile nav
- Wire up `mediator` auth
- Add some tests 
- QA

To check out the latest stitched into force see https://github.com/artsy/force/pull/3701. 

![notifications](https://user-images.githubusercontent.com/236943/54010524-f87b4600-4123-11e9-85cd-525fd132bb19.gif)
